siphon/simplewebservice/igra2.py

```diff
@@ -0,0 +1,427 @@
+# Copyright (c) 2013-2015 University Corporation for Atmospheric Research/Unidata.
```

Comment:
2018 only since you're adding this new file. FYI, we're about to replace MIT with BSD 3 (to match our other projects). We're also going to make this say "Copyright...University Corporation for Atmospheric Research/Unidata and Contributors" (or something like it). Nothing you need to do, but I thought you should be aware.